.github/workflows/test.yml

```diff
@@ -25,7 +25,8 @@ jobs:
       - name: Build and Check Package
         uses: hynek/build-and-inspect-python-package@v2.9.0
         with:
-          attest-build-provenance-github: 'true'
+          # Disabling because this is failing currently, see #466.
```

Comment:
@nicoddemus https://github.com/hynek/build-and-inspect-python-package/issues/105#issuecomment-2136262019

Reply:
To summarize, it is better then to use `with attestations` in the publish-pypi action instead of in `hynek/build-and-inspect-python-package`?

Reply:
Yeah, that's my stance. Though, not everybody feels that way.

FWIW, GitHub's own attestations are embedded into the UI. The PyPI ones are upload provenance, not necessarily build provenance, technically.